Source/Core/VideoCommon/AVIDump.cpp

```diff
@@ -323,6 +323,8 @@ void AVIDump::CloseVideoFile()
   av_frame_free(&s_scaled_frame);
 
   avcodec_free_context(&s_codec_context);
+
+  avio_closep(&s_format_context->pb);
```

Comment:
@iwubcode Good catch! If `CreateVideoFile` returns false (maybe because someone clicked "No, don't override my old video file"), then `CloseVideoFile` is called right away. In that case `s_format_context` isn't necessarily null, but it's definitely not valid data.

I'll add a check.